cartographer_ros/configuration_files/turtlebot.lua

```diff
@@ -22,6 +22,7 @@ options = {
   odom_frame = "odom",
   tracking_frame = "base_link",
   provide_odom = false,
```

Comment:
It occurs to me that this should be called "provide_odom_frame" instead. In that sense, it's actually not mutually exclusive with use_odom.